Makefile

```diff
@@ -381,8 +381,13 @@ ifdef LLAMA_BLIS
 endif # LLAMA_BLIS
 
 ifdef LLAMA_CUBLAS
-	MK_CPPFLAGS  += -DGGML_USE_CUBLAS -I/usr/local/cuda/include -I/opt/cuda/include -I$(CUDA_PATH)/targets/x86_64-linux/include -I/usr/local/cuda/targets/aarch64-linux/include
-	MK_LDFLAGS   += -lcuda -lcublas -lculibos -lcudart -lcublasLt -lpthread -ldl -lrt -L/usr/local/cuda/lib64 -L/opt/cuda/lib64 -L$(CUDA_PATH)/targets/x86_64-linux/lib -L/usr/local/cuda/targets/aarch64-linux/lib -L/usr/lib/wsl/lib
+	ifneq ('', '$(wildcard /opt/cuda)')
+		CUDA_PATH ?= /opt/cuda
+	else
+		CUDA_PATH ?= /usr/local/cuda
+	endif
```

Comment:
Wouldn't it be simpler to just use `-I` and `-L` multiple times for all possible locations and just change the order relative to master? Unless I'm missing something nonexistent directories are being ignored.

Reply:
The order doesn't matter, I just thought it was ugly that the default search location was split between the default CUDA_PATH and other -I/-L options that are added explicitly regardless of whether CUDA_PATH is set - this also means you can explicitly set CUDA_PATH to a nonexistent directory and have the build still succeed, which I don't like.

I would also be fine with using CUDA_PATH if set, otherwise falling back to a command that explicitly checks both /opt/cuda and /usr/local/cuda.

Reply:
Could this pr be merged? What other do I need to do?

Reply:
Sorry, I didn't mean to hold up the PR. In the first place, I would consider something like this only a minor detail.